splunk_add_on_ucc_framework/global_config.py

```diff
@@ -60,19 +60,26 @@ def from_dict(cls, **kwargs: Any) -> "OSDependentLibraryConfig":
 
 
 class GlobalConfig:
-    def __init__(self, global_config_path: str) -> None:
+    def __init__(
```

Comment:
Do you think we could cover tests and use cases to capture this alteration, to ensure that this isn't changed in future?

Reply:
It's indirectly tested by `from_file` calling it, do you think about some other tests we can have?

Reply:
I can add those in a separate small PR.